controller/label_blackbox_test.go

```diff
@@ -146,6 +146,9 @@ func (rest *TestLabelREST) TestUpdate() {
 		require.NotNil(t, jerrs)
 		require.Len(t, jerrs.Errors, 1)
 		require.Contains(t, jerrs.Errors[0].Detail, "version conflict")
+		ignoreString := "IGNORE_ME"
+		jerrs.Errors[0].ID = &ignoreString
+		compareWithGoldenUUIDAgnostic(t, filepath.Join(rest.testDir, "update", "conflict.golden.json"), jerrs)
```

Comment:
Please rename error golden files  according to this schema: `<reason>.errors.golden.json`.

Reply:
Done. 9ba9ad6a1f5473040108a9b847206eff65e1d1f2